gen/dcos-config-windows.yaml

```diff
@@ -63,7 +63,7 @@ package:
         ],
         "LocalFiles": [
             {
-                "Location": "{{ windows_dcos_var_path_json }}\\mesos\\mesos-agent.log",
+                "Location": "{{ windows_dcos_var_path_json }}\\log\\mesos\\mesos-agent.INFO.log",
```

Comment:
Remove "INFO" from the log line since it's not just INFO but all logging levels and also stderr?

Reply:
There is mesos-agent.log name convention for Logsync log in EE. Therefore removing INFO will cause some missunderstanding in EE - who is a origin of the log : NSSM or Logsync?
To workaround my suggestion is to use "-nssm" suffix: 
```
adminrouter-nssm.log
mesos-agent-nssm.log
dcos-diagnostics-nssm.log
```
Will it suit you, @vinodkone ?

Reply:
How are they differentiated on Linux? I think on Linux the log files here are call mesos.log etc.  What Logsync logs called on Linux? If that's not a thing in Linux, I would like to call these mesos.log and call the logsync logs something else. Basically, I want the naming to be consistent between Linux and Windows.

Reply:
@vinodkone 
The Logsync log on Linux named /var/log/mesos/mesos-agent.log. So this is inline. we just need distinguish NSSM now, which we are doing.